defaults/clade_emergence_dates.tsv

```diff
@@ -29,3 +29,4 @@ Nextstrain_clade	first_sequence
 22B (Omicron)	2021-12-01
 22C (Omicron)	2021-12-01
 22D (Omicron)	2022-04-01
+22E (Omicron)	2022-07-10
```

Comment:
The first BQ.1 could potentially have already appeared quite some time before July, first samples from Nigeria are early July, but given low sampling there it must have been around for a few months already.

Depends how tight we want to be. Global circulation started to become apparent early August.

<img width="1705" alt="image" src="https://user-images.githubusercontent.com/25161793/195433013-2010c5cd-b62f-4589-bf4c-4c6a5c63a210.png">